arlo_server/__init__.py

```diff
@@ -4,7 +4,8 @@
 from flask import Flask, jsonify, request, Response, redirect, session
 from flask_httpauth import HTTPBasicAuth
 
-from sampler import Sampler
+import sampler as sampler
```

Comment:
Isn't this equivalent to `import sampler`?

Reply:
yep! Will fix.